### PR TITLE
Writing a cross browser compatible date input

### PR DESCRIPTION
**Pull request by jaime-lynn**

### CODE REVIEW

**Comment by khalidabuhakmeh** [comment]
It would be nice to do a linkable codepen so folks can try out some code.

**Comment by jaime-lynn** [comment]
Oh, good idea! Let me throw this in WIP real quick and get something set up.

File: _posts/2019-10-24-writing-a-compatible-date-input.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+layout: post
+title: "Writing a Browser Compatible Date Input"
+date: 2019-10-24 13:00:00
+tags:
+- JavaScript
+- Vue
+- HTML
+categories:
+- JavaScript
+twitter_text: "Writing a Browswer Compatible Date Input"
```
Comment (kendaleiv) [inline _posts/2019-10-24-writing-a-compatible-date-input.md]:
```suggestion
twitter_text: "Writing a Browser Compatible Date Input"
```

Comment (jaime-lynn) [reply _posts/2019-10-24-writing-a-compatible-date-input.md]: How about `Writing a Bowser Compatible Date Input`? :smile:

Comment (kendaleiv) [reply _posts/2019-10-24-writing-a-compatible-date-input.md]: Save that one for April 1.

**Pull request merged**